analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/Extensions/IOperationExtensions.cs

```diff
@@ -18,88 +18,91 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-namespace SonarAnalyzer.SymbolicExecution.Roslyn
+namespace SonarAnalyzer.SymbolicExecution.Roslyn;
+
+internal static class IOperationExtensions
 {
-    internal static class IOperationExtensions
+    internal static ISymbol TrackedSymbol(this IOperation operation)
```

Comment:
Not related to this line specifically: how does symbol tracking and in general this feature play with inheritance (`override`) and shadowing (`new`)?
- virtual auto-property made non-auto in sub-class;
- virtual non-auto-property made auto in sub-class;
- virtual auto-property shadowed by non-auto in sub-class;
- virtual non-auto-property shadowed by auto in sub-class;
- multi-level inheritance, etc.

It would be worth adding some scenarios testing that.

Reply:
I added a check for `IsVirtual` and will add tests. Shadowing should not have any effect, but I will also add tests.